specs/experimental/op-contracts-manager.md

```diff
@@ -106,40 +102,43 @@ struct Roles {
     address challenger;
 }
 
+/// @notice The full set of inputs to deploy a new OP Stack chain.
 struct DeployInput {
     Roles roles;
     uint32 basefeeScalar;
     uint32 blobBasefeeScalar;
     uint256 l2ChainId;
-    bytes startingAnchorRoots;
+    // The correct type is OutputRoot memory but OP Deployer does not yet support structs.
+    bytes startingAnchorRoot;
     string saltMixer;
     uint64 gasLimit;
-    uint32 disputeGameType;
-    bytes32 disputeAbsolutePrestate;
+    // Configurable dispute game parameters.
+    GameType disputeGameType;
+    Claim disputeAbsolutePrestate;
     uint256 disputeMaxGameDepth;
     uint256 disputeSplitDepth;
-    uint64 disputeClockExtension;
-    uint64 disputeMaxClockDuration;
+    Duration disputeClockExtension;
+    Duration disputeMaxClockDuration;
```

Comment:
I think we should undo this whole diff: 
1. The comments feel like unnecessary implementation details
2. the custom types are not defined within this document, so either we need to define them here, or stick with elementary types.